Makefile

```diff
@@ -0,0 +1,99 @@
+# List all independent nodes, separated with escaped newlines
+NODES = \
+serial_bridge 
+
+# Code running onboard
+ONBOARD = onboard_tmp
+
+LCM_TYPES_DIR = biped_lcm
+LCM_PACKAGE_NAME = biped_lcm
+LCM_TARGET_JAR = biped_lcm.jar
+
+MATLAB_JAVA = 1.7
+
+# TODO: make this independent of DRAKE_DISTRO
+LCM_JAR = $(DRAKE_DISTRO)/build/install/share/java/lcm.jar
+
+#----------------------------------------------
+
+export CXX=g++
+export CXXFLAGS=-std=c++11 -Wall
+
+BUILD_DIR = build
+NODE_TARGET_NAME = main
+
+#path to this makefile, i.e. to the top level biped dir
+MKFILE_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+LCM_DEFS = $(wildcard $(LCM_TYPES_DIR)/*.lcm)
+
+default: all
```

Comment:
Can do, was thinking it might be easier to read with the variables near the rules they are relevant for